packages/components/src/constants.js

```diff
@@ -1,3 +1,8 @@
 const I18N_DOMAIN_COMPONENTS = 'tui-components';
+const CIRCULAR_PROGRESS_SIZE = {
+	small: 'small',
+	default: 'default',
+	large: 'large',
+};
 
-export default I18N_DOMAIN_COMPONENTS;
+export { I18N_DOMAIN_COMPONENTS, CIRCULAR_PROGRESS_SIZE };
```

Comment:
:scream: **Wow! Be careful!**
There is a lot of usage of `I18N_DOMAIN_COMPONENTS`.
That's why snapshots are updated...

Reply:
:D good catch, thanks